hevm-cli/hevm-cli.hs

```diff
@@ -316,10 +315,14 @@ equivalence cmd = do
       False -> do
         putStrLn "No discrepancies found"
         when (any isTimeout res) $ do
-          putStrLn "But timeout(s) occurred"
+          putStrLn "But timeout(s) occurred:"
```

Comment:
I think you may have forgotten to print the timeouts here? Probably a `<> (filter isTimeout res)` is missing here?